src/tap/grpc/server.rs

```diff
@@ -544,6 +544,11 @@ fn base_event<B, I: Inspect>(req: &http::Request<B>, inspect: &I) -> api::TapEve
             m.labels.insert("tls".to_owned(), tls);
             m
         }),
+        route_meta: inspect.route_labels(req).map(|labels| {
+            let mut m = api::tap_event::RouteMeta::default();
+            m.labels.extend(labels.as_ref().clone());
```

Comment:
I believe this will double clone the map. What about `m.labels.as_ref().iter()`?

Reply:
I _think_ the clone is necessary.  I get:

```
   --> src/tap/grpc/server.rs:549:22
    |
549 |             m.labels.extend(labels.as_ref().iter());
    |                      ^^^^^^ expected reference, found struct `std::string::String`
    |
    = note: expected type `(&std::string::String, &std::string::String)`
               found type `(std::string::String, std::string::String)`
```

So it could be `.iter().map(|(k, v)| (k.clone(), v.clone()))` ? which avoid copying the indexmap structure? Is that worth it?